modules/win_complus.py

```diff
@@ -0,0 +1,236 @@
+# -*- coding: utf-8 -*-
+'''
+Microsoft Component Service management via powershell module
+
+:platform:      Windows
+
+
+'''
+
+# Import python libs
+from __future__ import absolute_import
```

Comment:
from __future__ import unicode_literals

Please add the above to support different languages